internal/witness/cmd/witness/verifier/verifier.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package verifier is designed to verify the signatures produced by a witness.
+package verifier
+
+import (
+	"crypto"
+	"errors"
+	"fmt"
+
+	ct "github.com/google/certificate-transparency-go"
+	"github.com/google/certificate-transparency-go/internal/witness/api"
+	"github.com/google/certificate-transparency-go/tls"
+)
+
+// WitnessV consists of a CT signature verifier.
+type WitnessV struct {
```

Comment:
Maybe `Verifier` or `WitnessVerifier`?

I imagine this is going to need to go somewhere else since others will (eventually) use it? (maybe in `/internal/witness/verifier`?)

Reply:
Sure, changed to `WitnessVerifier` and moved it up (I knew it was in the wrong place but didn't know where the right one was :).